views/login.handlebars

```diff
@@ -1,14 +1,15 @@
-<section id="create-login" class='text-center'>
-
-  <div class='buddy-card p-0 w-100'></div>
-  <div class='buddy-card p-0 w-100'></div>
-
+<div class='signup-images container text-center'>
+  <img src='nav-logo.png' />
```

Comment:
same goes here: this src wont work. has to be relative and pointing at image folder so something like ./images/file